test/data/cuda_test_ignores.txt

```diff
@@ -0,0 +1,150 @@
+torch.ByteTensor.abs
```

Comment:
nit: I'd rename this test_cuda_ignores so it's obvious it only applies to test_cuda and (if it's easy) put a comment to that affect in the file.

Reply:
Good idea. Done